### PR TITLE
Assets copying, unify WORKING_DIRECTORY, fix linux build on some platforms

### PR DESCRIPTION
**Pull request by tuxuser**
* Copy `assets/` directory into <target dir> after build
* Use same `WORKING_DIRECTORY` for all platforms (same directory as binary)

This yields the following structure
```
bin/Debug-linux/
bin/Debug-linux/OpenNeighborhood
bin/Debug-linux/OpenNeighborhood/OpenNeighborhood
bin/Debug-linux/OpenNeighborhood/assets
bin/Debug-linux/OpenNeighborhood/assets/icons/*
bin/Debug-linux/OpenNeighborhood/assets/fonts/*
```

* Fix linux build by explicitly linking with libGL and libGLEW (fixed gentoo build for me, should not have any negative side-effects)

### CODE REVIEW

**Comment by ClementDreptin** [comment]
Hey tuxuser !

Thank you so much for contributing!
It seems like linking explicity with GLEW and OpenGL for linux breaks the remote build for the GitHub action (probably because they don't have the libraries installed on their machine). It should not be necessary anyway because the source code of `glad` is in `OpenNeighborhood/vendor/Glad` and it should take care of loading the OpenGL functions.

Can you please change `OpenNeighborhood/premake5.lua` accordingly? I'll put on comment on the related lines.

Thank you in advance!

Clément

File: OpenNeighborhood/premake5.lua [hunk 1]
```diff
@@ -54,7 +54,9 @@ project "OpenNeighborhood"
 			"X11",
 			"pthread",
 			"dl",
-			"stdc++fs"
+			"stdc++fs",
+			"GLEW",
+			"GL"
```
Comment (ClementDreptin) [inline OpenNeighborhood/premake5.lua:60]: Removing the comma from the first line and removing the next two lines should fix the build for the GitHub action, you shouldn't have to explicitly link to those libraries, that's weird.

Comment (ClementDreptin) [inline OpenNeighborhood/premake5.lua]: Removing the comma from the first line and removing the next two lines should fix the build for the GitHub action, you shouldn't have to explicitly link to those libraries, that's weird.

Comment (tuxuser) [reply OpenNeighborhood/premake5.lua]: We can try, on Gentoo the explicit linking was needed.

Comment (tuxuser) [reply OpenNeighborhood/premake5.lua]: Looks good ;)

Comment (ClementDreptin) [reply OpenNeighborhood/premake5.lua]: Thank you! I'm merging ;)

**Pull request merged**